keyregen/tarsnap-keyregen.1-man

```diff
@@ -26,7 +26,10 @@ and
 \fBtarsnap\fP(1).
 The term "compatible"
 here means that it is possible to re-encrypt archives stored with
-the first set of keys to be stored with the second set of keys.
+the first set of keys to be stored with the second set of keys.  This is
+required because Tarsnap has some keys which need to stay the same when
+re-encrypting data; otherwise, existing archives will become unreadable and
```

Comment:
Hmm, I don't think re-encrypting to completely different keys would make archives *unreadable*.  It would just screw up the deduplication.